cluster/images/hyperkube/master.json

```diff
@@ -37,6 +60,25 @@
               "--master=127.0.0.1:8080",
               "--v=2"
         ]
+    },
+    {
+      "name": "setup",
```

Comment:
Thanks, it also explains why I am consistently seeing 1 restart in the master pod. Seems like a _dirty_ approach though.